benches/bench_decimal.rs

```diff
@@ -0,0 +1,35 @@
+use criterion::{black_box, criterion_group, criterion_main, Bencher, Criterion};
+
+use pyo3::prelude::*;
+use pyo3::types::PyDict;
+#[cfg(feature = "rust_decimal")]
```

Comment:
All `#[cfg(feature = "rust_decimal")]` should be unnecessary due to the `required-features = ["rust-decimal"]` clause?

Reply:
I think you're right. I've removed it.